<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CharacteriseSimpleWidgetVerticalLayout</class>
 <widget class="QWidget" name="CharacteriseSimpleWidgetVerticalLayout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CharacteriseWidgetVerticalSimple</string>
  </property>
  <layout class="QHBoxLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout">
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="characterisation_gbox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="acceptDrops">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>Characterisation</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <item>
         <layout class="QVBoxLayout">
          <item>
           <layout class="QHBoxLayout">
            <item>
             <layout class="QHBoxLayout">
              <item>
               <widget class="QLabel" name="strat_comp_label">
                <property name="text">
                 <string>Strategy complexity:</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="start_comp_cbox">
                <item>
                 <property name="text">
                  <string>Single subwedge</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Few subwedges</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Many subwedges</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="rtwo_spacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout">
            <item>
             <layout class="QVBoxLayout">
              <item>
               <widget class="QCheckBox" name="account_rad_dmg_cbx">
                <property name="text">
                 <string>Account for radiation damage</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QCheckBox" name="optimised_sad_cbx">
                <property name="text">
                 <string>Optimised SAD</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="cone_spacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>1</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
